tomls/omnibus-arbitrum-sepolia/spot/dai.toml

```diff
@@ -17,7 +17,7 @@ args = [
     "<%= extras.synth_dai_market_id %>",
     "<%= extras.dai_oracle_id %>",
     "<%= extras.dai_oracle_id %>",
-    "<%= settings.dai_staleness_tolerance %>",
+    "<%= settings.big_cap_strict_staleness_tolerance %>",
```

Comment:
@kaleb-keny updated for all spot markets

Reply:
i'll setup stuff my end to keep on eye on these 😅

Reply:
thanks for pointing it out